src/parser.rs

```diff
@@ -140,15 +125,13 @@ fn search_for_attr(
         } else {
             match cur_node_value.kind() {
                 NODE_ATTR_SET => {
-                    if is_match {
-                        result.push((cur_node_value, path, real_depth));
-                    } else {
-                        let cur_node_casted = AttrSet::cast(cur_node_value).unwrap();
-                        cur_node_casted.entries().for_each(|entry| {
-                            stack.insert(0, (entry, path.clone()));
-                            remaining_items_cur += 1;
-                        });
-                    }
+                    let orig_stack_size = stack.len();
```

Comment:
if we don't need the offset, we also don't need to save the original length.

Reply:
Removed